infra/notifications/src/mobilePush.js

```diff
@@ -555,10 +455,86 @@ class MobilePush {
     await this._sendInChunks(data, deviceType, mobileRegisters)
   }
 
+  async _sendToCountryCode(data, deviceType, countryCode) {
```

Comment:
No need to fix now but maybe add a comment that if the server would get restarted in the middle of this method being called (for ex. due to a deploy), some notifications would not get sent.

Reply:
Will add that.